pages/science/species-thermo.rst

```diff
@@ -80,25 +80,14 @@ expressions to compute the properties, they all require thermodynamic property
 information for the individual species. For the phase types implemented at
 present, the properties needed are:
 
-1. the molar heat capacity at constant pressure :math:`\hat{c}^0_p(T)` for a
-   range of temperatures and a reference pressure :math:`P_0`;
-2. the molar enthalpy :math:`\hat{h}(T_0, P_0)` at :math:`P_0` and a reference
-   temperature :math:`T_0`;
-3. the absolute molar entropy :math:`\hat{s}(T_0, P_0)` at :math:`(T_0, P_0)`.
+1. the molar heat capacity at constant pressure :math:`\hat{c}^\circ_p(T)` for a
+   range of temperatures and a reference pressure :math:`p^\circ`;
+2. the molar enthalpy :math:`\hat{h}(T^\circ, p^\circ)` at :math:`p^\circ` and a reference
+   temperature :math:`T^\circ`;
+3. the absolute molar entropy :math:`\hat{s}(T^\circ, p^\circ)` at :math:`(T^\circ, p^\circ)`.
 
 See: :ref:`the Thermodynamic Models section <sec-thermo-models>`
```

Comment:
We probably don't need this link anymore since the very next section is the one being linked to 😄

Reply:
😂